cmd/snap/cmd_snap_op_test.go

```diff
@@ -39,6 +39,8 @@ type snapOpTestServer struct {
 	total   int
 }
 
+var _ = check.Suite(&SnapOpSuite{})
```

Comment:
go-check is a library which does not hack go test, so we need somehow to tell that you want to run a given suite. Any suggestions?